docs/guides/moodleapp/faq.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Moodle App FAQ
+sidebar_label: FAQ
+sidebar_position: 4
+tags:
+  - Moodle App
+---
+
+## How can I contribute to the Moodle App?
+
+You can help with [any issue marked with the "contribfriendly" label](https://tracker.moodle.org/browse/MOBILE-3231?jql=project%20%3D%20MOBILE%20AND%20status%20%3D%20Open%20AND%20labels%20%3D%20contribfriendly).
```

Comment:
Should this label, `contribfriendly` be added to the https://moodle.github.io/devdocs/general/development/tracker/labels page?

Reply:
I added the label in that page (in a new commit).

Reply:
Perfect! Thanks Dani!!